ql/ql/src/codeql_ql/style/MisspellingQuery.qll

```diff
@@ -11,7 +11,7 @@ predicate misspelling(string wrong, string right, string mistake) {
 }
 
 predicate additional_typos(string wrong, string right) {
-  wrong = "tranformer" and right = "transformer"
+  wrong = "transformer" and right = "transformer"
```

Comment:
```suggestion
  wrong = "tranformer" and right = "transformer"
```

This was an intentional typo.

Reply:
Sigh. I knew to look out for that file but missed reviewing it.